apps/web/.env.example

```
@@ -85,4 +85,8 @@ SENTRY_AUTH_TOKEN=
 
 # Analytics (if enabled)
 NEXT_PUBLIC_GA_ID=
-NEXT_PUBLIC_MIXPANEL_TOKEN=
\ No newline at end of file
+NEXT_PUBLIC_MIXPANEL_TOKEN=
+
+# PostHog Product Analytics
+# Uses /ingest reverse proxy (see next.config.ts rewrites)
+NEXT_PUBLIC_POSTHOG_KEY=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)